lib/Digitick/Sepa/TransferFile/Facade/CustomerCreditFacade.php

```diff
@@ -61,8 +61,15 @@ public function addTransfer($paymentName, array $transferInformation)
             $transferInformation['creditorName']
         );
         $transfer->setBic($transferInformation['creditorBic']);
-        $transfer->setRemittanceInformation($transferInformation['remittanceInformation']);
-
+        
+        if(array_key_exists('creditorReferenceInformation', $transferInformation) &&
+            strlen($transferInformation['creditorReferenceInformation'])>5 ) {
+                
+            $transfer->setCreditorReferenceInformation($transferInformation['creditorReferenceInformation']); 
+        }else{
```

Comment:
Is it really mutual exclusive that the remittance information can be set then there is no creditorReferenceInformation ?